src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj

```diff
@@ -134,6 +134,7 @@
       <ZipTargetFolder>..\..\zip\</ZipTargetFolder>
       <ZipToCreate>$(ZipTargetFolder)Particular.ServiceControl-$(GitVersion_MajorMinorPatch).zip</ZipToCreate>
     </PropertyGroup>
+    <Message Text="$(ZipTargetFolder)Particular.ServiceControl-$(GitVersion_MajorMinorPatch).zip" Importance="high"/>
```

Comment:
Does this have something to do with the build not completing?

Reply:
Or is this a debugging test?

Reply:
This is was (stupid I agree - the value outputed is already in the logs :P) debugging addition.